package.json

```diff
@@ -58,6 +58,7 @@
     "test:front:watch": "cross-env IS_EE=true run test:front --watch",
     "test:front:watch:ce": "cross-env IS_EE=false run test:front --watch",
     "test:generate-app": "yarn build:ts && node test/scripts/generate-test-app.js",
+    "test:generate-app:no:build": "node test/scripts/generate-test-app.js",
```

Comment:
I am only not sure if this is the right solution

Reply:
Yeah, unfortunately better options would take a lot more effort and considering this is just for internal tooling I don't think it's worth it to refactor generate-app just for this.

I would go with 'no-build' though, with a dash